editions/test/tiddlers/tests/files/tiddlywiki.files

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why is the tiddlywiki.files file needed?

Reply:
I think it's needed to pull the $:/core/macros/tabs tiddler into the test system. ... or is it available without it? I'll need to test that

Reply:
OK. It's not needed. I'll remove it